CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # CHANGELOG
 
 ## Unreleased
+- feat: Add `data` param to `ExceptionMechanism` class (#1450)
```

Comment:
I suggest to change this description to `Add support for sending arbitrary extra data as part of the exception mechanism`. I find this to be more helpful for readers than knowing the details of how this was implemented.

Reply:
No problem. I'm not a native speaker, so my descriptions might be weird :)